src/riak_core.erl

```diff
@@ -159,7 +159,13 @@ standard_join(Node, Ring, Rejoin, Auto) ->
                                                   node(),
                                                   gossip_vsn,
                                                   GossipVsn),
-            {_, Ring5} = riak_core_capability:update_ring(Ring4),
+            ParticipateInCoverage = app_helper:get_env(riak_core,participate_in_coverage),
```

Comment:
All this Ring2, Ring3, Ring4, Ring4a, Ring5, Ring6 might be nicer as a fold - but that's outside the scope of this change request.